test/Test/Indigo/Contracts/HomebaseLite/Utils.hs

```diff
@@ -36,27 +34,27 @@ deriving newtype instance Num Seconds
 
 deployContract
   :: MonadCleveland caps m
-  => m (Address, ContractHandle Parameter Storage ())
+  => m (ImplicitAddress, ContractHandle Parameter Storage ())
 deployContract = deployContract' id
 
 deployContract'
   :: MonadCleveland caps m
   => (Storage -> Storage)
-  -> m (Address, ContractHandle Parameter Storage ())
+  -> m (ImplicitAddress, ContractHandle Parameter Storage ())
 deployContract' = deployContractWithConf Nothing
 
 deployContractWithConf
   :: MonadCleveland caps m
   => Maybe FA2Config
   -> (Storage -> Storage)
-  -> m (Address, ContractHandle Parameter Storage ())
+  -> m (ImplicitAddress, ContractHandle Parameter Storage ())
 deployContractWithConf fa2conf modStor = do
   admin <- newAddress "admin"
-  let stor = modStor $ defaultStorage admin fa2conf
-  contract <- originateSimple "HomebaseLite" stor lorentzContract
+  let stor = modStor $ defaultStorage (MkConstrainedAddress admin) fa2conf
```

Comment:
You could use `toL1Address` here, too.

Reply:
Ugh..thought I covered all instances. Have fixed.